react-delegationdashboard/src/components/Overview/Cards/AutomaticActivationAction/index.tsx

```diff
@@ -1,9 +1,30 @@
+import ConfirmOnLedgerModal from 'components/ConfirmOnLedgerModal';
+import OwnerActionModal from 'components/Overview/OwnerActionModal';
+import { useContext } from 'context';
+import { DelegationTransactionType } from 'helpers/contractDataDefinitions';
+import { useDelegationWallet } from 'helpers/useDelegation';
 import React, { useState } from 'react';
-import AutomaticActivationModal from './AutomaticActivationModal';
 
 const AutomaticActivationAction = ({ automaticFlag }: { automaticFlag: string }) => {
+  const { ledgerAccount } = useContext();
   const [showAutomaticActivationModal, setShowAutomaticActivationModal] = useState(false);
+  const [showCheckYourLedgerModal, setShowCheckYourLedgerModal] = useState(false);
+  const [transactionArguments, setTransactionArguments] = useState(
+    new DelegationTransactionType('', '')
+  );
+  const { sendTransactionWallet } = useDelegationWallet();
 
+  const handleAutomaticActivation = () => {
+    let activation = Buffer.from(automaticFlag === 'true' ? 'false' : 'true').toString('hex');
```

Comment:
is it ok that it is reverted? right now, if `automaticFlag` is `true` then it will be parsed as `false`

Reply:
Yes, this is intended